pkgs/applications/editors/ne/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, ncurses, texinfo, tetex, perl, ghostscript }:
+
+
+stdenv.mkDerivation {
+  name = "ne-3.0.1";
+  version = "3.0.1";
+  src = fetchurl {
+    url = https://github.com/vigna/ne/archive/3.0.1.tar.gz;
+    md5 = "ec3dc82d10c271e95ecde630538f9f19";
+  };
+  buildInputs = [ ncurses tetex texinfo perl ghostscript ];
+  preBuild = ''
+    substituteInPlace src/makefile --replace "CC=c99" "cc=gcc"
```

Comment:
I tried and didn't work so I'm sticking with this solution for now, I'll try again before committing the changes